website/docs/networking/vpc-lattice/setup.md

```diff
@@ -0,0 +1,75 @@
+
+```bash
+$ PREFIX_LIST_ID=$(aws ec2 describe-managed-prefix-lists --query "PrefixLists[?PrefixListName=="\'com.amazonaws.$AWS_DEFAULT_REGION.vpc-lattice\'"].PrefixListId" | jq --raw-output .[])
+$ MANAGED_PREFIX=$(aws ec2 get-managed-prefix-list-entries --prefix-list-id $PREFIX_LIST_ID --output json  | jq -r '.Entries[0].Cidr')
+$ CLUSTER_SG=$(aws eks describe-cluster --name eks-workshop --output json| jq -r '.cluster.resourcesVpcConfig.clusterSecurityGroupId')
+$ aws ec2 authorize-security-group-ingress --group-id $CLUSTER_SG --cidr $MANAGED_PREFIX --protocol -1
+```
+
+Create a policy (`recommended-inline-policy.json`) in IAM with the following content that can invoke the gateway API and copy the policy arn for later use:
+
+```bash
+$ aws iam create-policy \
+--policy-name VPCLatticeControllerIAMPolicy \
+--policy-document file://workspace/modules/networking/vpc-lattice/controller/recommended-inline-policy.json
+```
+
+```file
+/networking/vpc-lattice/controller/recommended-inline-policy.json
+```
+
+Create the `system` namespace:
+```bash
+$ kubectl apply -f workspace/modules/networking/vpc-lattice/controller/deploy-namesystem.yaml
+```
+Retrieve the policy ARN:
+```bash
+$ export VPCLatticeControllerIAMPolicyArn=$(aws iam list-policies --query 'Policies[?PolicyName==`VPCLatticeControllerIAMPolicy`].Arn' --output text)
+```
+Create an `iamserviceaccount` for pod level permission:
+```bash
+$ eksctl create iamserviceaccount \
+    --cluster=${EKS_CLUSTER_NAME} \
+    --namespace=system \
+    --name=gateway-api-controller \
+    --attach-policy-arn=${VPCLatticeControllerIAMPolicyArn} \
+    --override-existing-serviceaccounts \
+    --region ${AWS_DEFAULT_REGION} \
+    --approve
+```
+
+---
+**CHOOSE AN INSTALLATION METHOD** for the controller between the two options below:
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; **Option 1:** *Install with kubectl*
+
+```bash
+$ kubectl apply -f workspace/modules/networking/vpc-lattice/controller/deploy-resources.yaml
+```
+      
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; **Option 2:** *Install with Helm*
+```bash
+$ aws ecr-public get-login-password --region us-east-1 | helm registry login --username AWS --password-stdin public.ecr.aws
+$ helm install gateway-api-controller \
+    oci://public.ecr.aws/aws-application-networking-k8s/aws-gateway-controller-chart\
+    --version=v0.0.7 \
+    --set=aws.region=${AWS_DEFAULT_REGION} --set=serviceAccount.create=false --namespace system
+```
+---
+
+Before creating a `Gateway`, we need to formalize the types of load balancing implementations with a [GatewayClass](https://gateway-api.sigs.k8s.io/concepts/api-overview/#gatewayclass).  
```

Comment:
This is one of the key module concepts. Adding some details about "types of load balancing implementations" would help connect the details of how things work. Link to existing docs would serve well too.

Reply:
changed